+dj/Header.m

```diff
@@ -81,7 +81,7 @@
             attrs.isNumeric = ~cellfun(@(x) isempty(regexp(sprintf('%s',x), ...
                 '^((tiny|small|medium|big)?int|decimal|double|float)', 'once')), attrs.type);
             attrs.isString = ~cellfun(@(x) isempty(regexp(sprintf('%s',x), ...
-                '^((var)?char|enum|date|year|time|timestamp)','once')), attrs.type);
+                '^((var)?char|enum|date|(var)?binary|year|time|timestamp)','once')), attrs.type);
```

Comment:
it's not labeling but an optional character sequence in regular expressions.  `(var)?char` is equivalent to `varchar|char`